assets/src/blocks/Articles/ArticlesFrontend.js

```diff
@@ -18,7 +18,9 @@ export const ArticlesFrontend = (props) => {
 
   const postId = !postIdClass ? null : postIdClass.split('-')[1];
 
-  const { posts, loadNextPage, hasMorePages, loading } = useArticlesFetch(props, postType, postId, document.body.dataset.nro);
+  const postCategorie = props.post_categorie || [];
```

Comment:
Hey I feel a bit lost in the review, I don't see any mention of `post_categorie` in the code (and because categor~y~ie is the singular for categories but in french, I assumed someone made a small mistake somewhere else that was copied here), do you know where it comes from/where it appears ?  
Also I have a bug in `PostTypeSelector` when I try to add a post type (`this.props.postTypes.find is undefined l.17`), but I don't think it's related to this PR.

Reply:
Hi @lithrel , 
Sorry for the trouble, I forgot to commit the [master theme changes](https://github.com/greenpeace/planet4-master-theme/pull/1204) :( 

Earlier we had server side render function to load articles block, but now we have api endpoints, so the server side code fails at [this point](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/master/classes/blocks/class-articles.php#L281) to fetch post categories and tags.

The workaround I added here is, pass post tags and categories as block attributes from [here](https://github.com/greenpeace/planet4-master-theme/pull/1204/files#diff-8e039a17b2aadb723e862bced701672b6e20a79e19ca6426b49084fc76d52624R87).